packages/workflows/__tests__/__system__/cli/List/activeWorkflows/cli.workflows.list.activeWorkflows.system.test.ts

```diff
@@ -0,0 +1,113 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ZosmfRestClient } from "../../../../../../rest";
+import { Session } from "@brightside/imperative";
+import { runCliScript, getUniqueDatasetName } from "../../../../../../../__tests__/__src__/TestUtils";
+import { ITestEnvironment } from "../../../../../../../__tests__/__src__/environment/doc/response/ITestEnvironment";
+import { ITestSystemSchema } from "../../../../../../../__tests__/__src__/properties/ITestSystemSchema";
+import { DeleteWorkflow, CreateWorkflow } from "../../../../..";
+import { TestProperties } from "../../../../../../../__tests__/__src__/properties/TestProperties";
+import { TestEnvironment } from "../../../../../../../__tests__/__src__/environment/TestEnvironment";
+import { Upload } from "../../../../../../zosfiles/src/api/methods/upload";
+import { ZosFilesConstants } from "../../../../../../zosfiles/src/api";
+import { join } from "path";
+import { IWorkflows } from "../../../../../src/api/doc/IWorkflows";
+
+let REAL_SESSION: Session;
+let testEnvironment: ITestEnvironment;
+let systemProps: TestProperties;
+let defaultSystem: ITestSystemSchema;
+let definitionFile: string;
+let wfKey: string;
+let system: string;
+let owner: string;
+let wfName: string;
+const workflow = join(__dirname, "../../../testfiles/demo.xml");
+
+describe("List workflow cli system tests", () => {
+    beforeAll(async () => {
+        testEnvironment = await TestEnvironment.setUp({
+            tempProfileTypes: ["zosmf"],
+            testName: "list_workflow_cli"
+        });
+        systemProps = new TestProperties(testEnvironment.systemTestProperties);
+        defaultSystem = systemProps.getDefaultSystem();
+        system = testEnvironment.systemTestProperties.workflows.system;
+        owner = defaultSystem.zosmf.user;
+        wfName = `${getUniqueDatasetName(owner)}`;
+        definitionFile = `${defaultSystem.unix.testdir}/${getUniqueDatasetName(owner)}.xml`;
+        REAL_SESSION = TestEnvironment.createZosmfSession(testEnvironment);
+    });
+
+    afterAll(async () => {
+        await DeleteWorkflow.deleteWorkflow(REAL_SESSION, wfKey);
+        await TestEnvironment.cleanUp(testEnvironment);
+    });
+    describe("List all workflows", () => {
+        beforeAll(async () => {
+            // Upload files only for successful scenarios
+            await Upload.fileToUSSFile(REAL_SESSION, workflow, definitionFile, true);
```

Comment:
Just curious, why does this file have to be uploaded in binary?

Reply:
Because of the UTF-8 encoding of the xml file causes an IZUWF0120E error.